tests/base_tests.py

```diff
@@ -585,7 +585,7 @@ def test_fit_not_changing_data_weighted(
     @pytest.mark.parametrize(
         "bad_weight_value, expected_message",
         [
-            (np.nan, "weight column must be non-null"),
```

Comment:
As in https://github.com/lvgig/tubular/pull/345#discussion_r1880067836

Reply:
pulled in the fix here :) https://github.com/lvgig/tubular/pull/351/commits/aed1e2c438dec42be95a2c1401c0648222165a2d